time-series/time-series-api/src/main/java/com/powsybl/timeseries/TimeSeriesTable.java

```diff
@@ -640,14 +679,32 @@ private void dumpCache(Writer writer, CsvConfig config, int point, CsvCache cach
         }
     }
 
-    public void writeCsv(Writer writer, char separator, ZoneId zoneId) {
+    private void writeTime(Writer writer, CsvConfig config, int point, int cachedPoint) throws IOException {
+        long time = tableIndex.getTimeAt(point + cachedPoint);
+        switch (config.getTimeFormat()) {
+            case DATE_TIME:
+                ZonedDateTime dateTime = ZonedDateTime.ofInstant(Instant.ofEpochMilli(time), ZoneId.systemDefault());
+                writer.write(dateTime.format(config.dateTimeFormatter));
+                break;
+            case FRACTIONS_OF_SECOND:
+                Long longTime = new Long(time);
+                double convertedTime = longTime.doubleValue();
+                writer.write(Double.toString(convertedTime / 1000));
```

Comment:
Is this code equivalent to:
```
writer.write(Double.toString(time / 1000.0));
```

I think that `time / 1000.0` will be automatically cast into a double (thanks to the `.0`)

Reply:
Done